### PR TITLE
building-PELUX-sources: remove image table

### PR DESCRIPTION
**Pull request by fhajredini**
The image table is hard to be read and often very confusing, this will
simplify it for new users.

Signed-off-by: Fisnik Hajredini <fhajredini@luxoft.com>

### CODE REVIEW

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 1]
```diff
@@ -53,29 +53,54 @@ Available images
...
+* core-image-pelux-minimal-dev-update 
+
+Qtauto-neptune Images
+"""""""""""""""""""""
+* core-image-pelux-qtauto-neptune
+* core-image-pelux-qtauto-neptune-dev
+* core-image-pelux-qtauto-neptune-update
+* core-image-pelux-qtauto-neptune-dev-update
+
+In case you want to build only the pelux base image, that can be done by removing qtauto from the variant name as it following.
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: remove it from "as it following"

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 2]
```diff
@@ -53,29 +53,54 @@ Available images
...
+For both ``intel-qtauto`` and ``rpi-qtauto`` variants, we support the following images: 
+
+Minimal Images
+""""""""""""""
+* core-image-pelux-minimal  
+* core-image-pelux-minimal-dev
+* core-image-pelux-minimal-update 
+* core-image-pelux-minimal-dev-update 
+
+Qtauto-neptune Images
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: Consider rename to:
Qt Automotive + Neptune 3 UI images

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 3]
```diff
@@ -53,29 +53,54 @@ Available images
...
+.. code-block:: bash
+
+    # For intel-qtauto:
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/intel-qtauto source sources/poky/oe-init-build-env build
+    # For rpi-qtauto:
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/rpi-qtauto source sources/poky/oe-init-build-env build
+
+For both ``intel-qtauto`` and ``rpi-qtauto`` variants, we support the following images: 
+
+Minimal Images
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst:101]: Replace capital I with i to images

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 4]
```diff
@@ -53,29 +53,54 @@ Available images
...
+.. code-block:: bash
+
+    # For intel: 
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/intel source sources/poky/oe-init-build-env build
+    # For rpi:
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/rpi source sources/poky/oe-init-build-env build
+
+For both ``intel`` and ``rpi`` variants, we support the following images: 
+
+Minimal Images
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: replace I to i in images

**Comment by JEderonn** [comment]
I have again read through the output for this PR on Jenkins, and I have still a few comments (sorry).

1. In wonder if you first should explain why there is a need with different build configuration variants (defined by VARIANT_NAME). Like the qt-auto variants which would add the Yocto layers with Qt Auto stuff.

2. Then, start to explain which (minimal) images that are available on the basic configuration, followed by the qt-auto configuration variants which adds the neptune images. This would perhaps remove the duplication of the part which explains which minimal images that are available.

My impression is that it's not clear why there is a need with different (build configuration) variants. What the configuration step does (e.g. adding Yocto layers for Qt and Qt Auto and more?), and how an end-user shall know which VARIANT_NAME to use if only minimal builds are of interest.

The different options for VARIANT_NAME is not described, please do.

What do you think?

**Comment by fhajredini** [comment]
I have further modified the page, added more information specifically regarding variants. Expecting feedback

**Comment by JEderonn** [comment]
I cannot find the build on Jenkins, would be nice to have it there. Don't understand why it's not.

**Comment by fhajredini** [comment]
> I cannot find the build on Jenkins, would be nice to have it there. Don't understand why it's not.

I think it wasnt rebased correctly. I just did it again, its currently building.

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 5]
```diff
@@ -22,76 +37,96 @@ image eventually, as the project evolves further and further away from vanilla
...
+Once you have synced the repo, run the following to configure the environment for Qt Automotive + Neptune 3 UI images. 
+
+.. code-block:: bash
+
+    # For intel-qtauto:
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/intel-qtauto source sources/poky/oe-init-build-env build
+    # For rpi-qtauto:
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/rpi-qtauto source sources/poky/oe-init-build-env build
+
+For both ``intel-qtauto`` and ``rpi-qtauto`` variants, we support the following images:
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: missing arp-qtauto

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 6]
```diff
@@ -22,76 +37,96 @@ image eventually, as the project evolves further and further away from vanilla
...
 .. code-block:: bash
 
+    # For intel: 
     TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/intel source sources/poky/oe-init-build-env build
+    # For rpi:
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/rpi source sources/poky/oe-init-build-env build
 
-The script will create configurations if there are no configurations present. A printout about
-creating ``conf/local.conf`` and ``conf/bblayers.conf`` is normal.
+For both ``intel`` and ``rpi`` variants, we support the following images:
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: missing arp

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 7]
```diff
@@ -22,76 +37,96 @@ image eventually, as the project evolves further and further away from vanilla
...
-When done fetching the sources, create a build directory and set up bitbake.
-``TEMPLATECONF`` tells the ``oe-init-build-env`` script which path to fetch
-configuration samples from.
+Once you have synced the repo, run the following to configure the environment for Qt Automotive + Neptune 3 UI images. 
+
+.. code-block:: bash
+
+    # For intel-qtauto:
+    TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/intel-qtauto source sources/poky/oe-init-build-env build
+    # For rpi-qtauto:
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst:94]: missing arp-qtauto for convenience

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 8]
```diff
@@ -22,76 +37,96 @@ image eventually, as the project evolves further and further away from vanilla
 PELUX. For example, a PELUX-based image has an empty root password and an ssh
 server installed by default.
 
-Building locally
-----------------
+Image variants
+--------------
+Pelux can be build for `intel`, `arp`, `rpi`, and `qemu`. Our variants are dependent on the target we are building for.
+If the variant contains only the target, a minimal image will be available to be built with no graphics layer. If the variant contains the target name, and the suffix ``-qtauto``, the minimal image will be available to be build as well as the image containing Qt Automotive + Neptune 3 UI.
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: Remove text '+ Neptune 3 UI"

File: docs/chapters/baseplatform/building-PELUX-sources.rst [hunk 9]
```diff
@@ -22,76 +37,96 @@ image eventually, as the project evolves further and further away from vanilla
...
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-qtauto-neptune-update     | :unavailable:`✘` | :available:`✔`   | :unavailable:`✘` | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-qtauto-neptune-dev-update | :unavailable:`✘` | :available:`✔`   | :unavailable:`✘` | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-
-When done fetching the sources, create a build directory and set up bitbake.
-``TEMPLATECONF`` tells the ``oe-init-build-env`` script which path to fetch
-configuration samples from.
+Once you have synced the repo, run the following to configure the environment for Qt Automotive + Neptune 3 UI images.
```
Comment (JEderonn) [inline docs/chapters/baseplatform/building-PELUX-sources.rst]: Remove text '+ Neptune 3 UI"

(enough in my opinion to mention neptune only in the context of specifying supported images)

**Comment by fhajredini** [comment]
I will merge this by the end of the day unless there are further comments!

**Pull request merged**